src/main/java/com/epam/emotionalhelp/service/impl/EmotionServiceImpl.java

```diff
@@ -17,6 +18,7 @@
 @Service
 @RequiredArgsConstructor
 public class EmotionServiceImpl implements EmotionService {
+    @Autowired
```

Comment:
No nee to use autowired here since we use  RequiredArgsConstructor annotation - it creates only one constructor for a class field